docs/src/main/sphinx/functions/ai.md

```diff
@@ -135,7 +186,8 @@ SELECT ai_gen('Describe Trino in a few words');
 :::
 
 :::{function} ai_mask(text, labels) -> varchar
-Masks values for the provided labels in the input text.
+Replaces values for the provided labels in the input text with the text 
```

Comment:
Maybe say "Mask the values ... by replacing ..."

Reply:
Updated